### ISSUE TITLE
Problem with lz4 compressed file "invalid RuntimePlatform"

### ISSUE THREAD

**Issue by oddMLan** [HearthSim/UnityPack#20]
https://puu.sh/sThVg/30d18719f4.zip

smilies.bundle fails with 
```pytb
Traceback (most recent call last):
  File "D:\Users\Kim\Documents\DuOS-shared\bundles\unityextract.py", line 146, in <module>
    main()
  File "D:\Users\Kim\Documents\DuOS-shared\bundles\unityextract.py", line 143, in main
    exit(app.run())
  File "D:\Users\Kim\Documents\DuOS-shared\bundles\unityextract.py", line 53, in run
    bundle = unitypack.load(f)
  File "D:\Python\Python35\lib\site-packages\unitypack\__init__.py", line 11, in load
    return env.load(file)
  File "D:\Python\Python35\lib\site-packages\unitypack\environment.py", line 21, in load
    ret.load(file)
  File "D:\Python\Python35\lib\site-packages\unitypack\assetbundle.py", line 40, in load
    self.load_unityfs(buf)
  File "D:\Python\Python35\lib\site-packages\unitypack\assetbundle.py", line 120, in load_unityfs
    asset.load(asset.data)
  File "D:\Python\Python35\lib\site-packages\unitypack\asset.py", line 94, in load
    self.tree.load(buf)
  File "D:\Python\Python35\lib\site-packages\unitypack\type.py", line 115, in load
    self.target_platform = RuntimePlatform(buf.read_uint())
  File "D:\Python\Python35\lib\enum.py", line 241, in __call__
    return cls.__new__(cls, value)
  File "D:\Python\Python35\lib\enum.py", line 476, in __new__
    raise ValueError("%r is not a valid %s" % (value, cls.__name__))
ValueError: 3392795225 is not a valid RuntimePlatform
```

If uncompressed before with UABE https://github.com/DerPopo/UABE
Unityextract.py can extract the assets just fine

**Comment by oddMLan** [comment]
I did install python-lz4, but nevertheless I'm still seeing this error message

**Comment by robert-nix** [comment]
Looks to be fixed as of 11c22bcc.
